index.js

```diff
@@ -188,7 +188,6 @@ PSPDFKitView.propTypes = {
      * fragmentTag: A tag used to identify a single PdfFragment in the view hierarchy.
      * This needs to be unique in the view hierarchy.
      *
-     * @platform android
```

Comment:
I think you accidentally removed this from the wrong doc. Should be removed from `onStateChanged`.

Reply:
Indeed! Nice Catch! Fixed.